src/backgroundreplacementprocessor/BackgroundReplacementVideoFrameProcessor.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { BackgroundReplacementOptions } from '..';
```

Comment:
nit: I recommend specifying the path for consistency with other import statements.